chips/nrf5x/src/timer.rs

```diff
@@ -98,6 +98,10 @@ impl Timer {
         self.timer().task_clear.set(1);
     }
 
+    pub fn set_bitmode(&self, bitmode: u32) {
```

Comment:
Document please? And maybe use an enum instead of a `u32`, since not all values are valid.